tests/dummy/app/templates/application.hbs

```diff
@@ -1,3 +1,42 @@
 <h2 id="title">Welcome to Ember</h2>
 
-{{outlet}}
\ No newline at end of file
+{{outlet}}
+<div>
+  <h5>Swappable</h5>
+  {{#shopify-swappable swapped=(action 'swapped')}}
+    {{#swappable-container as |c|}}
+      {{#each list as |item|}}
+        {{#swappable-item}}
+          {{item.name}}
+        {{/swappable-item}}
+      {{/each}}
+    {{/swappable-container}}
+    {{#swappable-container as |c|}}
+      {{#each listTwo as |item|}}
+        {{#swappable-item}}
+          {{item.name}}
+        {{/swappable-item}}
+      {{/each}}
+    {{/swappable-container}}
+  {{/shopify-swappable}}
```

Comment:
@bakerac4 Thoughts on streamlining the component invocation a bit with yielded components? And maybe rename the `shopify-*` components to have a little more meaning. Something like:

```hbs
{{#swappable-group swapped=(action 'swapped') as |group|}}
    {{#group.container as |c|}}
         {{#each list as |item|}}
             {{#c.item}} {{item.name}} {{/c.item}}
        {{/each}}          
    {{/group.container}}
    {{!-- .... --}}
{{/swappable-group}}
```

```hbs
{{#sortable-group sorted=(action 'sorted') as |group|}}
    {{!-- .... --}}
{{/sortable-group}}
```